HerPublicWebsite.BusinessLogic/Services/CsvFileCreator/CsvFileCreator.cs

```diff
@@ -272,6 +272,8 @@ public CsvRowReferralRequest(ReferralRequest request)
                 IncomeBand.GreaterThan31000 => "£31k or above",
                 IncomeBand.UnderOrEqualTo34500 => "Below £34.5k",
                 IncomeBand.GreaterThan34500 => "£34.5k or above",
+                IncomeBand.UnderOrEqualTo36000 => "£36k or below",
+                IncomeBand.GreaterThan36000 => "Above £36k",
```

Comment:
Leon said he'd get this confirmed for us at stand-up earlier, so hold on merging until resolved.